book/src/proposals/broadmoor.md

```diff
@@ -470,42 +477,74 @@ For example:
 
 By classifying these risky features, A/B Street can keep tabs as people in the
 simulation are exposed to these risks. You can follow an individual's trip and
-see what risks they are exposed to. For a macro view, you can aggregate and
+see what risks they are exposed to, or, for a macro view, you can aggregate and
 identify map-wide hot spots for specific problems. Using A/B Street, you can
 compare how a proposal affects these metrics.
 
 <div class="wrap-two">
 
 <div class="media-box">
 <h4 class="chart-header">Identifying Hotspots</h4>
-<img class="chart" src="../assets/broadmoor/montlake-520-hotspot.png" />
+<img class="chart" 
+  src="../assets/broadmoor/montlake-520-hotspot.png" 
+  alt="a heatmap covering about a square mile of streets south of the Montlake
+  Bridge, with hotspots on the intersections leading up to the bridge"/>
+
+A/B Street's "problem" view helps you identify risk exposure hotspots.
+
+These streets south of the Montlake Bridge exhibit several risk factors,
+including wide streets with fast moving vehicles and many-legged intersections.
 
-South of the Montlake bridge, crossing Highway 520, has complex arterial
-intersections _and_ relatively high bike and foot traffic, meaning many people
-are exposed to these risks.
+Because the Montlake Bridge is one of only a couple plausible connections
+between the University District and central Seattle, it has relatively high bike
+and foot traffic.
+
+The inherent riskiness of these _kinds_ of streets, paired with the high
```

Comment:
I love how you explain this -- there's potential problems inherent to the geometry here, and people are forced to bottleneck through it, so it's an area worthy of consideration.